### PR TITLE
Add patch 5.2 abilities

### PR DESCRIPTION
**Pull request by ryan-dyer-sp**
@doxxx for review

### CODE REVIEW

File: app/js/ffxivcraftmodel.js [hunk 1]
```diff
@@ -245,9 +245,6 @@ function ApplyModifiers(s, action, condition) {
         control += (0.2 * s.effects.countUps[AllActions.innerQuiet.shortName]) * s.synth.crafter.control;
     }
 
-    // Since game version 5.0, effects increasing control are capped at crafter's original control + 3000
-    control = Math.min(control, s.synth.crafter.control + 3000);
```
Comment (doxxx) [inline app/js/ffxivcraftmodel.js]: Why is this being removed?

File: app/js/ffxivcraftmodel.js [hunk 2]
```diff
@@ -279,8 +276,8 @@ function ApplyModifiers(s, action, condition) {
         progressIncreaseMultiplier += calcNameOfElementsBonus(s);
     }
 
-    if (AllActions.innovation.shortName in s.effects.countDowns) {
```
Comment (doxxx) [inline app/js/ffxivcraftmodel.js]: Has the effects of Innovation changed?

File: app/js/ffxivcraftmodel.js (hunk 1 above)
Comment (tanpro260196) [reply app/js/ffxivcraftmodel.js]: Inner quiet no longer cap at 3000.

File: app/js/ffxivcraftmodel.js (hunk 2 above)
Comment (tanpro260196) [reply app/js/ffxivcraftmodel.js]: Yes:
Effect changed from **"Increases efficiency of Synthesis and Touch actions by 20% for the next four steps"** to **"Increases efficiency of Touch actions by 50% for the next four steps."**

**Comment by doxxx** [comment]
Deployed.

**Pull request merged**